packages/synapse-react-client/src/components/CardDeck/CardDeckCardProps.ts

```diff
@@ -0,0 +1,11 @@
+import { ReactNode } from 'react'
+
+export type CardDeckCardProps = {
+  title: string /* title of card */
+  description: string /* description of card */
+  ctaButtonText: string /* call to action button text */
+  ctaButtonURL: string /* call to action button URL */
+  titleIcon?: ReactNode /* icon to be displayed on the card */
+  headerImage?: ReactNode /* header image to be displayed on the card */
```

Comment:
Only real change was to change this from a fileHandleAssociation to a ReactNode (i.e. directly pass the image instead of a Synapse-specific pointer to the image)